GOVERNANCE.md

```diff
@@ -0,0 +1,47 @@
+# Tern Governance
+
+This document defines the project governance for Tern.
+
+## Overview
+
+**Tern**, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building a high-quality and high performance container inspection tool. The community is governed by this document with the goal of defining how community should work together to achieve this goal.
+
+## Community Roles
+
+* **Users:** Members that engage with the Contour community via any medium (Slack, GitHub, mailing lists, etc.).
+* **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+* **Maintainers**: The Tern project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for Tern.
+
+### Maintainers
+
+New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the other maintainers.
+
+### Supermajority
+
+A supermajority is defined as two-thirds of members in the group.
```

Comment:
There are currently only two maintainers. How can we get to a supermajority without a third one?